task_1_1_1__/HeapSort.java

```diff
@@ -0,0 +1,52 @@
+
+import java.util.*;
+import java.util.Arrays;
+
+public class HeapSort {
```

Comment:
Classes should be placed in base package instead of root package.

For example, you can create package `ru.nsu.fit.<your_second_name>.heapsort`.

Base package should be same for both src classes and test classes, but in separate src roots (idea provides src/main and src/test roots by default)